server.go

```diff
@@ -81,5 +87,6 @@ func handleComments(w http.ResponseWriter, r *http.Request) {
 func main() {
 	http.HandleFunc("/comments.json", handleComments)
 	http.Handle("/", http.FileServer(http.Dir("./public")))
+	log.Println("Server started: http://localhost:3000")
```

Comment:
Yea, though it logs that it fails. Perhaps I can say "Starting server…" but it's not a huge deal.

```
2015/03/01 12:00:04 Server started: http://localhost:3000
2015/03/01 12:00:04 listen tcp :3000: bind: address already in use
exit status 1
```